Nos/Views/Components/Author/AuthorListView.swift

```diff
@@ -36,6 +41,22 @@ struct AuthorListView: View {
         .nosNavigationBar(title: .localizable.mention)
         .onAppear {
             isSearching = true
+
+            guard let relatedAuthors = relatedAuthors else { return }
+            filteredAuthors = relatedAuthors
+        }
+        .onChange(of: searchController.authorResults) { _, newValue in
+            filteredAuthors = []
+
+            guard let currentAuthor = currentUser.author else {
+                filteredAuthors = newValue
+                return
+            }
+            let sortedAuthors = newValue.sortByMutualFollowees(with: currentAuthor)
+            filteredAuthors += sortedAuthors
+
+            guard let relatedAuthors = relatedAuthors else { return }
+            filteredAuthors += relatedAuthors
```

Comment:
Ah, ok. I saw an unexpected order when I was testing this earlier, and I assumed it was because sortedAuthors is sorted by itself, then added to filteredAuthors, then relatedAuthors is sorted by itself, then added to filteredAuthors. So the final filteredAuthors array isn't sorted by number of mutual followees, and you might get something like this:

https://github.com/user-attachments/assets/8f4817d9-779e-419c-98d0-3fb617dd7087

I'm not sure that's entirely what's happening here, but as you can see in the video, the authors aren't sorted by mutual followees.

Reply:
You are right, this is exactly what is happening:

>  I assumed it was because sortedAuthors is sorted by itself, then added to filteredAuthors, then relatedAuthors is sorted by itself, then added to filteredAuthors

But isn't this supposed to be expected if you have typed something in the search bar? Like your search should come above any authors who are in the thread regardless of mutual followers?

Reply:
ooh, looks like the video is broken, because I only saw the first 2 seconds like a picture. I will look into it and see if I find anything strange.

Reply:
It wasn't the video broken, it was a bad internet, my bad!

Reply:
I just did some debug printing and the mutual followees are listed according to the count. It looks like the function for the mutualFollowees and the knownFollowers are not doing the same thing currently 🤔

Reply:
@joshuatbrown , I removed the mutual followees sorting algorithm.